webvtt/parsers.py

```diff
@@ -177,25 +179,39 @@ def _parse_cue_block(self, block):
                         raise MalformedCaptionError(
                             '{} in line {}'.format(e, block.line_number + line_number))
                 else:
-                    raise MalformedCaptionError(
-                        '--> found in line {}'.format(block.line_number + line_number))
+                    additional_blocks = self._compute_blocks(
+                        ['WEBVTT', '\n'] + block.lines[line_number:]
+                    )
+                    break
+                    # TODO: Is there a time we still might want this?
```

Comment:
Hey @sontek I had the same question. Initially I thought it would be good that the user is aware of the issues in the file. Take a permissive approach (fix things where we can) and leave the file to parse even with some errors seems better overall. I think the library is too strict at the moment: for example we raise an exception if we find a caption text missing a timeframe line. Maybe we could ignore it as probably video players are doing. What's your take on this?

Reply:
I think for now, ignoring it is the right approach.  I don't think we want people to have to manually fix every file they run across